.github/workflows/cherryPick.yml

```diff
@@ -26,9 +26,28 @@ jobs:
 
   createNewVersion:
     needs: validateActor
+    runs-on: ubuntu-latest
     if: ${{ fromJSON(needs.validateActor.outputs.IS_DEPLOYER) && github.event.inputs.NEW_VERSION == '' }}
-    uses: Expensify/App/.github/workflows/createNewVersion.yml@main
-    secrets: inherit
+    outputs:
+      NEW_VERSION: ${{ steps.getNewVersion.outputs.NEW_VERSION }}
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          ref: main
+
+      - name: Create new BUILD version
+        uses: Expensify/App/.github/actions/javascript/triggerWorkflowAndWait@main
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          WORKFLOW: createNewVersion.yml
```

Comment:
I see that `SEMVER_LEVEL` is still an input of `createNewVersion.yml`. To clarify, it's fine not to pass it because it's `required: false`, right?

Reply:
Yep! That's one of the change I've decided to optimistically keep from the other PR